fundingmanager.go

```diff
@@ -552,13 +552,22 @@ func (f *fundingManager) start() error {
 			if chanType.IsSingleFunder() && chanType.HasFundingTx() &&
 				channel.IsInitiator {
 
-				err := f.cfg.PublishTransaction(
+				var fundingTxBuf bytes.Buffer
+				err := channel.FundingTxn.Serialize(&fundingTxBuf)
+				if err != nil {
+					fndgLog.Errorf("Unable to serialize "+
+						"funding transaction %v: %v",
+						channel.FundingTxn.TxHash(), err)
+				}
+
+				err = f.cfg.PublishTransaction(
 					channel.FundingTxn, "",
 				)
 				if err != nil {
 					fndgLog.Errorf("Unable to rebroadcast "+
-						"funding tx for "+
+						"funding tx %x for "+
 						"ChannelPoint(%v): %v",
+						fundingTxBuf.Bytes(),
```

Comment:
what gets printed if we errored above?

Reply:
Probably the bytes that were written to the buffer before the serialization error occurred. Maybe we should reset the buffer in that case so the error here would just read: `Unable to rebroadcast funding tx  for ChannelPoint(...): ...`.

Reply:
Done.